packages/oruga/src/components/breadcrumb/Breadcrumb.vue

```diff
@@ -0,0 +1,130 @@
+
+/**
+ * The classic breadcrumb, in different colors, sizes, and states
+ * @displayName Breadcrumb
+ * @style _breadcrumb.scss
+ */
+defineOptions({
+    isOruga: true,
+    name: "OBreadcrumb",
+    configField: "breadcrumb",
+});
+
+const props = defineProps({
+    /** Override existing theme classes completely */
+    override: { type: Boolean, default: undefined },
+    /**
+     * Color variant of the control
+     * @values primary, info, success, warning, danger, and any other custom color
+     */
+    variant: {
+        type: String,
+        default: () => getOption("breadcrumb.variant"),
+    },
+    /**
+     * Size of the control
+     * @values small, medium, large
+     */
+    size: {
+        type: String,
+        default: () => getOption("breadcrumb.size"),
+    },
+    align: {
+        type: String,
+        default: getOption("breadcrumb.size"),
+    },
+    // separator
+    separator: {
+        type: String,
+        default: () => getOption("breadcrumb.size"),
+    },
+    tag: {
+        type: String,
+        default: () => getOption("breadcrumb.tag"),
+    },
+    // -----------------------------------------------------------------
+    /**
+     * This is used internally
+     * @ignore
+     */
+    // class props (will not be displayed in the docs)
+    /** Class of the root element */
+    rootClass: {
+        type: [String, Array, Function] as PropType<ComponentClass>,
+        default: undefined,
+    },
+    /** Class of the button elements wrapper */
+    wrapperClass: {
+        type: [String, Array, Function] as PropType<ComponentClass>,
+        default: undefined,
+    },
+    /** Class of the button size */
+    sizeClass: {
+        type: [String, Array, Function] as PropType<ComponentClass>,
+        default: undefined,
+    },
+    /** Class of the button variant */
+    variantClass: {
+        type: [String, Array, Function] as PropType<ComponentClass>,
+        default: undefined,
+    },
+    /** Class of the button separator */
+    separatorClass: {
+        type: [String, Array, Function] as PropType<ComponentClass>,
+        default: undefined,
+    },
+    /** Class of the button align */
+    alignClass: {
+        type: [String, Array, Function] as PropType<ComponentClass>,
+        default: undefined,
+    },
+});
+
+// --- Computed Component Classes ---
+const computedTag = computed(() => {
+    return props.tag ? props.tag : "ul";
+});
+// --- Computed Component Classes ---
+
+const rootClasses = defineClasses(
+    ["rootClass", "o-breadcrumb"],
+    [
+        "sizeClass",
+        "o-breadcrumb--",
+        computed(() => props.size),
+        computed(() => !!props.size),
+    ],
+    [
+        "variantClass",
+        "o-breadcrumb--",
+        computed(() => props.variant),
+        computed(() => !!props.variant),
+    ],
+    [
+        "separatorClass",
+        "o-breadcrumb--",
+        computed(() => props.separator),
+        computed(() => !!props.separator),
```

Comment:
What is the separator class for? I would think of a class on the separator element, but not on the root with a suffix.